### PR TITLE
[stable/insights] FWI-3207 - add rbac secret mgnt

### PR DESCRIPTION
**Pull request by vitorvezani**
**Why This PR?**
Add secret management (get, create, delete) for repo-scan-job service-account

Fixes internal FWI-3207

**Changes**
Changes proposed in this pull request:

* add get/create/delete secrets to repo-scan-job service-account

**Checklist:**

* [X] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [X] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

**Review by jdesouza — APPROVED** [review]

**Comment by vitorvezani** [comment]
> ## Fairwinds Insights - Auto-scan job has failed ❌
> 
> > [View the Full Report](https://insights.fairwinds.com/gh/FairwindsOps/charts/vv%2Fadd-rbac-secret-mgnt%2FFWI-3207)
> 
> Show Logs

re-run fixed it

File: stable/fairwinds-insights/templates/rbac-repo-scan-job.yaml [hunk 1]
```diff
@@ -25,6 +25,9 @@ rules:
 - apiGroups: [""]
   resources: ["events"]
   verbs: ["list", "get"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["create", "delete", "get"]
```
Comment (rbren) [inline stable/fairwinds-insights/templates/rbac-repo-scan-job.yaml]: do we need `get`? That one is kinda scary

Comment (vitorvezani) [reply stable/fairwinds-insights/templates/rbac-repo-scan-job.yaml]: I don't think we need it - I just checked... we create and delete based on known secret name so...

Comment (vitorvezani) [reply stable/fairwinds-insights/templates/rbac-repo-scan-job.yaml]: Done, removed `get` from the verbs

**Review by rbren — APPROVED** [review]

**Pull request merged**